example/immutable.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const Errors = require('../Errors.js');
+
+let piKindOf = 22/7;
+
+function getItRight(){
+    if(typeof piKindOf != 'undefined'){
+        const err = new Errors.Immutable;
+        err.setMessage(
+            'piKindOf',
+            'global'
+        );
+        throw err;
+    }
+    piKindOf = Math.PI;
+}
+
+getItRight();
```

Comment:
I think you should change this from being an example of a global object.

Global objects are always defined unless you explicitly define them as undefined. It's a confusing example.